core/shared/src/main/scala/zio/metrics/Metrics.scala

```diff
@@ -0,0 +1,80 @@
+package zio.metrics
+
+import zio.{IO, Trace, UIO}
+
+import java.io.IOException
+
+final case class Metrics private[zio] (metrics: Set[MetricPair.Untyped]) extends Serializable {
+
+  /**
+   * Gets all current metrics in pretty debug view.
+   */
+  def prettyPrint(implicit trace: Trace): UIO[String] =
+    MetricsRenderer.prettyPrint(metrics)
+
+  /**
+   * Dumps all current metrics to the console.
+   */
+  def dump(implicit trace: Trace): IO[IOException, Unit] =
+    prettyPrint.flatMap(zio.Console.print(_))
+
+}
+
+private[metrics] object MetricsRenderer {
```

Comment:
At this point this could be changed to `private object Metrics` or these could be inlined as private methods in the `Metrics` class.